Source/Core/Core/NetPlayCommon.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <SFML/Network/Packet.hpp>
+
+#include <array>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+
+namespace NetPlay
+{
+constexpr u32 LZO_IN_LEN = 1024 * 64;
```

Comment:
Does this actually need to be exposed publicly?

Reply:
It doesn't now I suppose.